Comment API Updated

araldortogul opened the pull request:
The updates need to be merged into master branch.

haticerk reviewed (changes requested):
I have some suggestions to improve the experience. Other than that, the external API and tests work fine.

* Maybe you can use @berfinsimsekk's post model to get post ID. So, the practice-app can work more similar to real-app we will design.
* Also, please remove the migrations files except the __init__.py one, to not to create conflicts with new changes with user model.

Well done!

araldortogul commented:
Thank you for your valuable comments @haticerk. I have updated the comments such that the parent posts are from @berfinsimsekk's **Post API**. Now, I believe the API looks more like the real app that we are going to design.
I have also removed the migration files!

Please let me know if you detect a bug.

haticerk reviewed (approved):
Everything looks great!
